hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/api/TestNSSummaryEndpointWithFSO.java

```diff
@@ -943,7 +943,7 @@ private void setUpMultiBlockKey() throws IOException {
         BUCKET_ONE_OBJECT_ID,
         VOL_OBJECT_ID,
         Collections.singletonList(locationInfoGroup),
-        getBucketLayout());
+        getBucketLayout(), FILE7_SIZE_WITH_REPLICA);
```

Comment:
Should this be the actual data size instead of the size with replicas? Same for other changes.

Reply:
Yes! Thanks for the good-catch!! I found I have to update some of the key-size calculation in those test files to make the test result be able to align with the expectation values. updated!

Reply:
The original key-size calculation in these tests didn't consider the replication type of the key, which didn't reflect the actual size of key with replica. Thus, updated the way to calculate the key size in these tests!